client/src/ledger_app_clients/ethereum/client.py

```diff
@@ -671,3 +678,13 @@ def provide_proxy_info(self, payload: bytes) -> RAPDU:
         for chunk in chunks[:-1]:
             self._exchange(chunk)
         return self._exchange(chunks[-1])
+
+    def sign_eip7702_authorization(self,
+                                   bip32_path: str,
+                                   delegate: bytes,
+                                   nonce: int,
+                                   chain_id: Optional[int] = None) -> RAPDU:
+        return self._exchange_async(self._cmd_builder.sign_eip7702_authorization(bip32_path,
```

Comment:
This could create a Python object like this one : https://github.com/LedgerHQ/app-ethereum/blob/25a18425c1a7a11b5497fa8dfe30a5eee66f6083/client/src/ledger_app_clients/ethereum/proxy_info.py
And in the builder function you'd just call its `serialize` method.

Reply:
Will do